extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -148,15 +154,15 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 				);
 				books.push(book);
 			} catch (e) {
-				vscode.window.showErrorMessage(localize('openConfigFileError', 'Open file {0} failed: {1}',
-					path.join(root, '_config.yml'),
-					e instanceof Error ? e.message : e));
+				let error = e instanceof Error ? e.message : e;
```

Comment:
Is there a helper function for this in the tests? We usually have a getErrorMessage floating around somewhere.